recipes/omsl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "omsl" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ mslversion }}
+
+source:
+  url: {{ mslsourceurl }}
+  sha256: {{ mslsourcesha256 }}
+
+build:
+  script_env:
+   - MSLVERSION={{ mslversion }}
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
Why is python listed in the host requirements, the install script is only copying files, it doesn't look like python is used at all?

Reply:
Correct, I will remove it.

Reply:
Same question for pip. Why is pip needed to install these libraries?